### ISSUE TITLE
Design issue while adding `showLeadingZeros = {true} `.

### ISSUE THREAD

**Issue by ParthPatwalaTA** [wojtekmaj/react-date-picker#101]
```showLeadingZeros = {true}``` property addes 0 outside of input type. This causes design issues. 
```
<DatePicker 
value={this.state.date} 
onChange= {(value) => this.setState({ date: value })}
showLeadingZeros = {true}
/> 
```

**Comment by wojtekmaj** [comment]
Yep, it's kind of hacky. There's no way to display leading zeros in number inputs though, so they are only "decorative".
Is there a specific issue I could help you with?

**Comment by ParthPatwalaTA** [comment]
So in my code, it is causing a design issue. Date is displaying kind of like  'o1 -o9 - 2019'. Generating extra space and all. Because if we apply design to the input field then it won't affect to '0'. So it's font-size, color and spacing will not change.
How is it working in [this](http://projects.wojtekmaj.pl/react-date-picker/) demo? Additional 0 is properly set and there is no spacing issue and other design issue.

**Comment by ParthPatwalaTA** [comment]
Also can you put 'react-date-picker__button__input__day', 'react-date-picker__button__input__month' and 'react-date-picker__button__input__year' in their seperate div and add 0 to their respective div? This way we can apply width and design for particular div elements.

**Comment by pjsvis** [comment]
I have a similar issue. When I use showLeadingZeroes={true} I do not see the leading zeroes. See CodeSandbox example here https://codesandbox.io/s/oqlo635xq6

**Comment by ParthPatwalaTA** [comment]
Updated issue for showLeadingZeroes. #107 #108

**Comment by wojtekmaj** [comment]
@pjsvis 
1. You're using react-date-picker version so old it didn't have `showLeadingZeros` flag
2. You have a typo in flag name

By the way, there's [React-DateRange-Picker](/wojtekmaj/react-daterange-picker) too!

**Comment by wojtekmaj** [comment]
Fixed in 209321477b8c15274df9e4021ff33bfee5bc06b5 by wrapping zero in a separate span element.

**Comment by ParthPatwalaTA** [comment]
Check this out. https://github.com/wojtekmaj/react-date-picker/pull/108#issuecomment-428855123
